packages/apiserver/src/utils/helpersFunctions/groupByKey.ts

```diff
@@ -0,0 +1,13 @@
+export const groupByKey = <T>(items: T[], key: string): Record<number, T[]> => {
```

Comment:
we can move out this generic type and use it in other modules without need to duplicate `Record<number, Person[]>` etc

Reply:
Added GroupRecord<T> type